app/managers.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self.conn = sqlite3.connect("cinema.sqlite")
+    
+    def create(self, first_name: str, last_name: str):
+        self.conn.execute("INSERT INTO actors (first_name, last_name)"
+                       "VALUES (?, ?)",
+                       (first_name, last_name))
+        self.conn.commit()
+        self.conn.close()
+
+    def all(self):
+        cursor = self.conn.cursor()
+        cursor.execute("SELECT * FROM actors")
+        return [Actor(*row) for row in cursor.fetchall()]
+    
+    def update(self, actor):
+        cursor = self.conn.cursor()
+        cursor.execute("UPDATE actors SET name = ?, age = ? WHERE id = ?", (actor.name, actor.age, actor.id))

```

Comment:
The `UPDATE` SQL statement uses incorrect field names 'name' and 'age'. It should use 'first_name' and 'last_name' to match the database schema for the `actors` table.